itest/starboard_cli_test.go

```diff
@@ -64,6 +64,91 @@ func NewPodSpec(podName string, containers map[string]string, args ...string) (*
 	return pod, err
 }
 
+var _ = Describe("Starboard CLI dryrun", func() {
```

Comment:
I believe that we should keep **Starboard CLI** as top level Description and add these new `It` cases under **Starboard CLI / Command init**. There's already `It("should initialize Starboard")` and you're exercising it with the --dry-run flag so we'd better keep the tests together.

So our goal is that all specs related to the init command read as follows:
- Starboard CLI / Command init / should initialize Starboard
- Starboard CLI / Command init / should return an error if the wrong dryrun flag is provided
- Starboard CLI / Command init / should return an error if dryrun flag provides invalid parameters
- Starboard CLI / Command init /should show which resources will be created

Reply:
That was my plan, although I wasn't able to achieve that because there was a before step that was performing the "init". Although, it is probably my lack of ginkgo knowledge... I will have another crack at it.

Reply:
Ah I see. If it's getting too complex I'm okey with your initial implementation... I forgot that we call init and cleanup command around each test case ...